kube/services/ssjdispatcher/ssjdispatcher-deploy.yaml

```diff
@@ -60,6 +60,11 @@ spec:
               configMapKeyRef:
                 name: manifest-global
                 key: dispatcher_job_num
+          - name: JOB_IMAGES
+            valueFrom:
```

Comment:
Hey,
You should make your own section in the `manifest.json`:
```
{ 
   "global" : { ... },
  "ssjdispatcher": { ... }
}
```
, then reference the `manifest-ssjdispatcher` configmap ...

Also - make the environment variable optional, so it still works if that configmap does not exist.

Also - go ahead and prepare a PR against `cdis-manifest` with the new config ...

Reply:
Made the environment variable optional in 9f12e9d02e4a210e6b176d444b8764550de07714